.github/CODEOWNDERS

```diff
@@ -0,0 +1,2 @@
+#All Files/PRS
```

Comment:
filename is incorrect / typo.

might also want to add dependabot / renovate users.